src/ui/initiativeUtils.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import removeMd from "remove-markdown";
+import {type TimestampMs} from "../util/timestamp";
+import {type NodeAddressT} from "../core/graph";
+import {type InitiativeWeight} from "../plugins/initiatives/initiative";
+import {type NodeWeight} from "../core/weights";
+
+// TODO: create formal initiative type once shape is defined (see github PR #1864)
+export type InitiativeEntry = {|
+  +id: string, // GUID
+  +title: string,
+  +timestampMs: TimestampMs,
+  +weight: InitiativeWeight,
+  +completed: boolean,
+  // user nodes
+  +champions: $ReadOnlyArray<NodeAddressT>,
+  +dependencies: $ReadOnlyArray<NodeAddressT>,
+  // an activity node
+  +references: $ReadOnlyArray<NodeAddressT>,
+  +contributions: $ReadOnlyArray<ContributionEntry>,
+|};
+
+type ContributionEntry = {|
+  // GUID
+  +key: string,
+  // Title is required, as this is essential for attribution.
+  +title: string,
+  // Defaults to an empty array.
+  +contributors: $ReadOnlyArray<NodeAddressT>,
+  // Timestamp of this node, but in ISO format as it's more human friendly.
+  +timestampMs: TimestampMs,
+  // Defaults to null.
+  +weight: NodeWeight,
+|};
+
+export const getPlainDescFromMd = ({description}: {description: string}) =>
+  removeMd((description: string));
```

Comment:
Don't think we need the type cast here
```suggestion
  removeMd(description);
```